ui/src/styles/index.scss

```
@@ -0,0 +1,66 @@
+}
+
+html,
+body,
+#root {
+  margin: 0;
+  padding: 0;
+  height: 100%;
+}
+
+body {
+  font-family: "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)